vue-components/tests/a11y/AllComponents.spec.ts

```diff
@@ -151,7 +151,8 @@ describe( 'OptionsMenu', () => {
 			{ label: 'potato', description: 'root vegetable' },
 			{ label: 'duck', description: 'aquatic bird' },
 		];
-		const wrapper = mount( OptionsMenu, { props: { menuItems } } );
+		const lookupLabel = 'Label of the parent Lookup';
+		const wrapper = mount( OptionsMenu, { propsData: { menuItems, label: lookupLabel } } );
```

Comment:
```suggestion
		const wrapper = mount( OptionsMenu, { props: { menuItems, label: lookupLabel } } );
```

https://test-utils.vuejs.org/migration/#propsdata-is-now-props